Description: increase tests timeout to make them succeed on mips
Author: Dejan Latinovic <Dejan.Latinovic@imgtec.com>
Forwarded: not-yet
Last-Update: 2015-01-09

--- a/tests/auto/api/tst_api.cpp
+++ b/tests/auto/api/tst_api.cpp
@@ -576,7 +576,7 @@
     qbs::Project project = setupJob->project();
     const QScopedPointer<qbs::BuildJob> buildJob(project.buildAllProducts(qbs::BuildOptions()));
     QTimer::singleShot(1000, buildJob.data(), SLOT(cancel()));
-    QVERIFY(waitForFinished(buildJob.data(), 300000));
+    QVERIFY(waitForFinished(buildJob.data(), 600000));
 }
 
 void TestApi::infiniteLoopBuilding_data()
@@ -595,7 +595,7 @@
     QScopedPointer<qbs::SetupProjectJob> setupJob(qbs::Project().setupProject(setupParams,
                                                                               m_logSink, 0));
     QTimer::singleShot(1000, setupJob.data(), SLOT(cancel()));
-    QVERIFY(waitForFinished(setupJob.data(), 300000));
+    QVERIFY(waitForFinished(setupJob.data(), 600000));
     QVERIFY2(setupJob->error().toString().toLower().contains("cancel"),
              qPrintable(setupJob->error().toString()));
 }
--- a/tests/auto/blackbox/tst_blackbox.cpp
+++ b/tests/auto/blackbox/tst_blackbox.cpp
@@ -106,7 +106,7 @@
     QProcess process;
     process.setProcessEnvironment(params.environment);
     process.start(cmdLine);
-    const int waitTime = 5 * 60000;
+    const int waitTime = 10 * 60000;
     if (!process.waitForStarted() || !process.waitForFinished(waitTime)) {
         m_qbsStderr = process.readAllStandardError();
         if (!params.expectFailure)
